src/test/c_api_tests.cpp

```diff
@@ -236,10 +237,79 @@ TEST_F(CapiInferencePreparationTest, Basic) {
     //////////////////
     //  INFERENCE
     //////////////////
+    // we need to prepare server now TODO
+    // remove when C-API start implemented
+    std::string port = "9000";
+    randomizePort(port);
+    char* argv[] = {
+        (char*)"OpenVINO Model Server",
+        (char*)"--model_name",
+        (char*)"dummy",
+        (char*)"--model_path",
+        (char*)"/ovms/src/test/dummy",
+        (char*)"--port",
+        (char*)port.c_str(),
+        (char*)"--log_level",
+        (char*)"DEBUG",
+        nullptr};
+
+    ovms::Server& server = ovms::Server::instance();
+    std::thread t([&argv, &server]() {
+        ASSERT_EQ(EXIT_SUCCESS, server.start(9, argv));
+    });
+    auto start = std::chrono::high_resolution_clock::now();
+    while ((ovms::Server::instance().getModuleState("GRPCServerModule") != ovms::ModuleState::INITIALIZED) &&
+           (std::chrono::duration_cast<std::chrono::seconds>(std::chrono::high_resolution_clock::now() - start).count() < 1)) {
+    }
+    std::this_thread::sleep_for(std::chrono::seconds(1));  // TODO
+    OVMS_InferenceResponse* response = nullptr;
+    OVMS_Server* cserver = reinterpret_cast<OVMS_Server*>(&server);
+    status = OVMS_Inference(cserver, request, &response);
+    ASSERT_EQ(nullptr, status);
+    // verify GetOutputCount
+    uint32_t outputCount = -1;
+    status = OVMS_InferenceResponseGetOutputCount(response, &outputCount);
+    ASSERT_EQ(nullptr, status);
+    ASSERT_EQ(outputCount, 1);
+    // verify GetParameterCount
+    uint32_t parameterCount = -1;
+    status = OVMS_InferenceResponseGetParameterCount(response, &parameterCount);
+    ASSERT_EQ(nullptr, status);
+    ASSERT_EQ(0, parameterCount);
```

Comment:
We should have test with at least one parameter

Reply:
There is one:
https://github.com/openvinotoolkit/model_server/pull/1524/files#diff-afb4aa495986273b9abea2e2606d0ae255e2d81d17e539eb6a53e506b582c494R471

But I had to create mocked inference response to have such parameter added.